src/common/components/Abilities/Abilities.tsx

```diff
@@ -0,0 +1,27 @@
+import classnames from "classnames/bind";
+import React from "react";
+import { CharacterSelector } from "../../redux/Character.slice";
+import { useSelector } from "../../redux/reduxHooks";
+import styles from "./Abilities.module.css";
+import { AbilityRow } from "./AbilityRow/AbilityRow";
+
+const cx = classnames.bind(styles);
+const { useState, useEffect } = React;
+
+export const Abilities = () => {
+  const baseCharacter = useSelector(CharacterSelector);
+  const { abilities, modifiers, proficiencyBonus = 0 } = baseCharacter;
```

Comment:
This is a useful shortcut (I learned today) in order to pull specific properties off an object. It makes it a little cleaner. Down below I can just call proficiencyBonus instead of baseCharacter.proficiencyBonus.